### DISCUSSION TITLE
03_pytorch_computer_vision.ipynb crossEntropyLoss()

### DISCUSSION THREAD

**Discussion by Jiammmmmin** [mrdbourke/pytorch-deep-learning#1028] [Q&A]
![image](https://github.com/user-attachments/assets/cd09ea3f-3aaf-45a2-8e40-5abad78dca0c)
![image](https://github.com/user-attachments/assets/9154271b-7407-4464-a576-20553241b92a)

it seems that I need to convert the `y` to `torch.tensor(y)`, but it does not help. Anyone else having the same problem as me?

**Comment by axuanyaa** [comment]
please set your y up to me ?

**Comment by rithvikshetty** [comment]
What tensor did you convert it to? I think it should be `torch.LongTensor`
